### ISSUE TITLE
Is it possible to use this with off the shelf trackers (like pebblebee)

### ISSUE THREAD

**Issue by jstride** [biemster/FindMy#71]
Use case: insert location into home assistant or view location on android device/web 

Reason: icloud doesn't show trackers such as pebblebee.

**Comment by biemster** [comment]
No, this does not work with off the shelf trackers, only custom firmware ones. They are very easy (and fun!) to make however, I'd suggest to start with either the ch592 or tlsr825x as they are cheap and widely available.
